packages/web-components/src/components/va-radio-option/va-radio-option.tsx

```diff
@@ -56,19 +56,24 @@ export class VaRadioOption {
 
   render() {
     const { checked, ariaDescribedby, name, value, label } = this;
+    const id = this.el.id || (name + value);
     return (
       <Host
         aria-checked={checked ? `${checked}` : 'false'}
         aria-describedby={(checked && ariaDescribedby) || null}
-        aria-label={name}
```

Comment:
Removed this `aria-label` as it prevented the actual content in the label & description from being read out by a screen reader. Thanks @joshkimux!